src/pages/dashboard/settings/Plans.vue

```diff
@@ -155,6 +160,75 @@ const hightLights = computed<Stat[]>(() => ([
     value: currentPlanSuggest.value?.name,
   },
 ]))
+
+const supabase = useSupabase()
+
+async function getUsage() {
+  const apps = await supabase.from('apps')
+    .select('app_id')
+    .eq('user_id', main!.user!.id!)
+
+  if (!apps.data)
+    return
+
+  const usage = await supabase.from('app_usage')
+    .select('mau, storage, bandwidth')
+    .gte('created_at', main.cycleInfo?.subscription_anchor_start)
+    .lte('created_at', main.cycleInfo?.subscription_anchor_end)
+    .in('app_id', apps.data.map(app => app.app_id))
+
+  const plan = plans.value.find(p => p.name === 'Pay as you go')!
+
+  let totalMau = 0
+  let totalStorage = 0
+  let totalBandwidth = 0
+
+  usage?.data?.forEach((item) => {
+    totalMau += item.mau
+    totalStorage += item.storage / (1024 * 1024 * 1024) // Convert bytes to gigabytes
+    totalBandwidth += item.bandwidth / (1024 * 1024 * 1024) // Convert bytes to gigabytes
+  })
+
+  const payg_base = {
+    mau: plan.mau,
+    storage: plan.storage,
+    bandwidth: plan.bandwidth,
+  }
+
+  const payg_units = {
+    mau: plan.mau_unit,
+    storage: plan.storage_unit,
+    bandwidth: plan.bandwidth_unit,
+  }
+
+  const basePrice = plan.price_m
+
+  const totalPrice = computed(() => {
+    const mauPrice = totalMau > payg_base.mau ? (totalMau - payg_base.mau) * payg_units!.mau! : 0
+    const storagePrice = totalStorage > payg_base.storage ? (totalStorage - payg_base.storage) * payg_units!.storage! : 0
+    const bandwidthPrice = totalBandwidth > payg_base.bandwidth ? (totalBandwidth - payg_base.bandwidth) * payg_units!.bandwidth! : 0
```

Comment:
Can you invert the condition and put the 0 first? It would make it more readable; I'm seeing it on a tiny screen, and it required some re-reads to understand where the condition finishes and where the processing starts